Cargo.toml

```diff
@@ -26,8 +26,8 @@ env_logger = "0.8.2"
 
 [profile.release]
 panic = "abort"
-codegen-units = 1
-lto = "fat"
+#codegen-units = 1
+#lto = "fat"
```

Comment:
These gain about 5% performance, but make it a hassle to test things because of much longer compile times. I think it is better to have them off by default